src/In.ProjectEKA.HipService/Link/LinkPatient.cs

```diff
@@ -40,7 +40,7 @@ public class LinkPatient
         public virtual async Task<ValueTuple<PatientLinkEnquiryRepresentation, ErrorRepresentation>> LinkPatients(
             PatientLinkEnquiry request)
         {
-            var (patient, error) = PatientAndCareContextValidation(request);
+            var (patient, error) = await PatientAndCareContextValidation(request);
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6044012)